src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/StoredProcedures/AcquireExportJobs/acquireExportJobs.js

```diff
@@ -145,6 +145,6 @@ function acquireExportJobs(maximumNumberOfConcurrentJobsAllowedInString, jobHear
     }
 
     function throwTooManyRequestsError() {
-        throw new Error(ErrorCodes.RequestEntityTooLarge, `The request could not be completed.`);
+        throw new Error(429, `The request could not be completed.`);
```

Comment:
>429 [](http://example.com/codeflow?start=24&length=3)

ErrorCodes.TooManyRequests sound more appropriate than a const

Reply:
This const isn't in provided out of the box :( http://azure.github.io/azure-cosmosdb-js-server/DocDbWrapperScript.js.html

I could create our own, but it is still defined per proc